src/inet/IPAddress.cpp

```diff
@@ -45,6 +45,14 @@ namespace Inet {
 
 IPAddress IPAddress::Any;
 
+IPAddress::IPAddress(const IPAddress & other)
```

Comment:
Can we do

`IPAddress::IPAddress(const IPAddress & other) = default;
`
?

Reply:
I admit that I was not sure if I should use ```default``because it implies C++11. Is that OK to state that C+11 is the minimum requirement for Chip ?

Reply:
I think we're already compiling as c++11.

Reply:
Done.